src/utils/challenges/challenge.js

```diff
@@ -34,7 +34,7 @@ async function generateChallenge(type, opts, ctx = {}, wait = false) {
     ctx.token = token;
   } catch (error) {
     if (error.message === '429') {
-      const { email } = this.config.validation;
+      const { mailto: email } = this.config.validation;
       const duration = moment().add(opts.throttle, 'seconds').toNow(true);
       const msg = `We've already sent you an email, if it doesn't come - please try again in ${duration} or send us an email`;
```

Comment:
I know this isn’t required for your change, but I suggest we include the email address if the message, like:
```javascript
  const msg = `... or send us an email at ${supportEmail}`;
```